manifests/cf-manifest/test-config/smoke_tests.yml

```diff
@@ -0,0 +1,18 @@
+---
+api: api.((system_domain))
+apps_domain: ((app_domain))
+user: "admin"
+password: ((secrets_uaa_admin_password))
+org: "SMOKE_TESTS"
+space: "SMOKE_TESTS"
+use_existing_org: false
+use_existing_space: false
+skip_ssl_validation: false
+artifacts_directory: "/tmp/artifacts"
+suite_name: CF_SMOKE_TESTS
+logging_app: ''
+runtime_app: ''
+ginkgo_opts: ''
+backend: ''
+admin_user: ((admin_user))
```

Comment:
The cf-smoke-tests project only have user/password fields. I created a commit that you can cherry-pick to fix this: 7fc6994a7a450cd8aca9eba69e2a85a8a45837c9.

(I stumbled into this one as part of https://www.pivotaltracker.com/story/show/155630096)

Reply:
👍